### ISSUE TITLE
"sounds" or "audio"?

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/limelight#108]
previously we have always talked about "audio" media; i notice that now it is "sounds". just wondering if there is a reason for this change & whether it would be worth having a bit of discussion about what the difference is?
i think "sounds" is fine for sound effects, but we also upload music clips, voice clips, etc that maybe "audio" is a more encompassing word for.
but then, as we will also be having audio & video streams, maybe there is reason to differentiate between something that is pre-recorded or live ... ???

**Comment by rohrlach** [comment]
We can certainly change the label.  The reason I went with sounds is that audio is an uncountable noun so the convention (in terms of the code) gets confusing

**Comment by helenvarley** [comment]
ok, if there's a code reason that's fine, but for the interface audio might make more sense. @vickis do you have an opinion about it?

**Comment by rohrlach** [comment]
It may eventually not even matter, considering #109

**Comment by helenvarley** [comment]
yes, i think it's a minor point, but what we have learned is that it's very important to be consistent in terminology, so that for example the user manual is clear, and there is no misunderstanding with multiple developers etc. so it's just a matter of making a decision.
